Build warning for esp32 on `tests/kernel/ipi_cascade/kernel.ipi_cascade.smp`

**Describe the bug**
Building `tests/kernel/ipi_cascade/kernel.ipi_cascade.smp` on `m5stickc_plus/esp32/procpu` produces the following warning:

```
/home/ddegrasse/work/zephyr/zephyr-upstream/zephyr/soc/espressif/esp32/esp32-mp.c:221:9: warning: implicit declaration of function 'z_sched_ipi' [-Wimplicit-function-declaration]
  221 |         z_sched_ipi();
```

**To Reproduce**
1. west build -p always -b m5stickc_plus/esp32/procpu -T tests/kernel/ipi_cascade/kernel.ipi_cascade.smp
2. See build warning

**Expected behavior**
Build should pass without warning

**Impact**
Failures in weekly twister run, see here: https://github.com/zephyrproject-rtos/zephyr/actions/runs/14812886367

--- a/soc/espressif/esp32/esp32-mp.c
+++ b/soc/espressif/esp32/esp32-mp.c
@@ -22,6 +22,7 @@
 #ifdef CONFIG_SMP
 
 #include <ipi.h>
+#include <ksched.h>
 
 #ifndef CONFIG_SOC_ESP32_PROCPU
 static struct k_spinlock loglock;

